apps/server/src/modules/learnroom/service/group-deleted-handler.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Group, GroupDeletedEvent } from '@modules/group';
+import { Injectable } from '@nestjs/common';
+import { EventsHandler, IEventHandler } from '@nestjs/cqrs';
+import { Course } from '../domain';
+import { CourseDoService } from './course-do.service';
+
+@Injectable()
+@EventsHandler(GroupDeletedEvent)
+export class GroupDeletedHandlerService implements IEventHandler<GroupDeletedEvent> {
```

Comment:
Dont know if there is a arc specification but i would prefer those event handlers and events in the same folder or directly next to each other